cypress/e2e/Customer/Create_Cloud/Login-NT.cy.js

```diff
@@ -0,0 +1,123 @@
+describe('Login NT Flow ', () => {
+    beforeEach(() => {
+        cy.on('uncaught:exception', (err, runnable) => {
+            return false
+        })
+
+        cy.wait(1000)
+
+        cy.intercept({
+            url: 'https://web-test.bangmod.cloud/auth/login',
+            method: 'GET',
+        }).as("User")
+
+        cy.visit('https://web-test.bangmod.cloud/auth/login')
+    })
+
+    it('Action success', () => {
+
+            cy.get('#username').type('qa@gmail.com');
+            cy.get('#password').type('123456');
+            cy.get('.btn').click();
+
+
+            cy.get('.d-flex > :nth-child(1)').eq(1).type('1');
+            cy.get('.d-flex > :nth-child(2)').type('2');
+            cy.get('.d-flex > :nth-child(3)').type('3');
+            cy.get('.d-flex > :nth-child(4)').type('4');
+            cy.get('.d-flex > :nth-child(5)').type('5');
+            cy.get('.d-flex > :nth-child(6)').type('6');
+
+
+    })
+
+    it('Validation (User does not enter Username and Password. The system display alert message “Please input data”)', () => {
+
+
+            cy.get('.btn').click();
+            cy.get('.text-danger').contains('Please input data')
+
+    })
+
+        it('Usabilities (User click “Sign up“ is a hyperlink. The system leads to Registration page.)', () => {
+
+            cy.get('[href="#"]').click();
```

Comment:
do not use # to make the test pass
do it right by using the real href **https://web-test.bangmod.cloud/registration**